server.py

```diff
@@ -8,7 +9,35 @@ def desk():
     return render_template("index.html")
 
 
-# def ask(word):#
+@app.route("/tagged/word/%string word%")

```

Comment:
But why? I asked you for the wildcard for `word` parameter, which type is`str`. What does mean `%string word%`?
```suggestion
@app.route("/tagged/word/<string:word>")
```
See documentation here: http://exploreflask.com/en/latest/views.html#url-converters